docs/DocGenerator/config.yaml

```diff
@@ -10,11 +10,17 @@ Include paths:
     path: "../../tests/integration/test_vulnerability_detector"
 
 Include regex:
-  - "^test_.*py$|readme.md$"
+  - "^test_.*py$"
+
+Group files:
+  - "README.md"
+
+Function regex:
+  - "^test_"
 
 Ignore paths:
   - "../../tests/integration/test_wazuh_db/data"
-  - "*/data/*"
+  - "/data/*"
```

Comment:
Are you checking the directories level by level? How is exactly used this regex?

Reply:
These 3 options are examples of how the Ignore path can be implemented. All of them have different levels of impact.